crates/swc_ecma_parser/src/parser/stmt.rs

```diff
@@ -748,7 +748,11 @@ impl<'a, I: Tokens> Parser<I> {
         })
     }
 
-    fn parse_var_declarator(&mut self, for_loop: bool) -> PResult<VarDeclarator> {
+    fn parse_var_declarator(
+        &mut self,
+        for_loop: bool,
+        kind: &VarDeclKind,
```

Comment:
I think we don't need to use reference here, as `VarDeclKind` implements `Copy`.